### PR TITLE
Replacevaluesbugfix

### PR DESCRIPTION
**Pull request by AlexSananka**

### CODE REVIEW

**Comment by AlexSananka** [comment]
@africanmathsinitiative/developers this is ready for review

**Comment by lilyclements** [comment]
@AlexSananka would you mind explaining to me what the bug was that you fixed? :)

**Comment by AlexSananka** [comment]
This were minor developer errors on the panel.
The first one was if you click rdonewvalue and then close and re open it has a developer error(that's fixed).
The second one was when you click rdooldmissing close and re open you have a developer error(that's is fixed too).
The third one was when you click either of the rdos in the ucrpnl and close then reopen they will reset it go back to the default one which is rdomissing.This is now sorted.

**Comment by lilyclements** [comment]
Great thanks! I'll review this now

**Comment by AlexSananka** [comment]
That's sad.I'll try fix that.

**Comment by AlexSananka** [comment]
@lilyclements have a test on this.

**Comment by lilyclements** [comment]
Reviewing this now

File: instat/dlgReplaceValues.vb [hunk 1]
```diff
@@ -137,6 +150,7 @@ Public Class dlgReplaceValues
 
         clsReplace.SetRCommand(frmMain.clsRLink.strInstatDataObject & "$replace_value_in_data")
         clsReplace.AddParameter("old_value", "-99", iPosition:=3)
+        clsReplace.RemoveParameterByName("old_is_missing")
```
Comment (dannyparsons) [inline instat/dlgReplaceValues.vb]: This can't be needed because you are resetting the function at the top of this sub.

**Comment by AlexSananka** [comment]
@dannyparsons This is ready again for you or @africanmathsinitiative/developers someone can test this it's ready for re-review.

**Pull request merged**